db/re/skill_db.yml

```diff
@@ -42856,15 +42854,15 @@ Body:
         Time: 30000
     Duration2:
       - Level: 1
-        Time: 5500
+        Time: 3500
       - Level: 2
-        Time: 8500
+        Time: 5500
       - Level: 3
-        Time: 11500
+        Time: 8500
       - Level: 4
-        Time: 14500
+        Time: 11500
       - Level: 5
-        Time: 17500
+        Time: 14500
```

Comment:
This looks like it should be 3s*level, but I'm just guessing (also you substraced 2000ms for level 1, but 3000ms for all other levels.